common/http.go

```diff
@@ -207,6 +207,12 @@ func (c *DatabricksClient) checkHTTPRetry(ctx context.Context, resp *http.Respon
 		// In this case don't retry and return the original error from httpclient
 		return false, err
 	}
+	if resp.StatusCode == 429 {
+		return true, APIError{
+			ErrorCode: "TOO_MANY_REQUESTS",
+			Message:   "Current request has to be retried",
+		}
+	}
```

Comment:
Awesome! This makes it so much easier if its handled at the client level rather than retrying at the resource levels. I think in most places having a linear retry with jitter should be enough. Do not think any of them would require exponential backoff.

Reply:
whatever is standard is the best :)